hexrd/ui/overlays/__init__.py

```diff
@@ -61,7 +68,12 @@ def default_overlay_refinements(overlay_type):
     if overlay_type not in default_refinements:
         raise Exception(f'Unknown overlay type: {overlay_type}')
 
-    return copy.deepcopy(default_refinements[overlay_type])
+    refinements = copy.deepcopy(default_refinements[overlay_type])
+
+    # Only give it the required indices
+    indices = unitcell._rqpDict[material.unitcell.latticeType][0]
```

Comment:
Shouldn't we grad the parameter list from the `PowderCalibrator.params` (which are in this case, just the reduced lattice params), or `PowderCalibrator.full_params` (which is the full concatenated list, [instr params] + [reduced lattice params])

Reply:
@psavery ...